functions.php

```diff
@@ -100,6 +100,18 @@ function getUser($nick, $conn = null, $client = null) {
   return $user;
 }
 
+function deleteUser($nick, $conn = null, $client = null) {
+  if ($conn) {
+    $sql = "delete from users where login = '$nick'; ";
```

Comment:
`sprintf(...)` is not appropriate for escaping SQL literals and identifiers as it doesn't handle SQL quoting/escaping semantics. The preparing of the concatenated SQL does nothing to make it safer.

As I believe this is PHP app is using PDO, you should be using `$stmt.bindParam(...)`: http://php.net/manual/en/pdostatement.bindparam.php